angular/README.md

```diff
@@ -1,37 +1,59 @@
 # Angular
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14. A bazel build was then configured alongside the Angular CLI using the Angular Architect tooling - the same tooling used by the CLI used within bazel. This shows how a project can be in a "hybrid mode" where some developers (and maybe the CI system) can use Bazel, but others can continue using their familiar tools.
```

Comment:
You don't want to keep this info about cli-style vs bazel-style along with the link to `../angular-ngc`?

Reply:
good catch, I should restore some signposts here